.travis.yml

```diff
@@ -18,5 +18,8 @@ branches:
 install:
  - true
 
+before_script:
+  - _JAVA_OPTIONS="-Xmx1024m -Xms128m"
```

Comment:
If this is needed for test, why don't we configure it in the *.gradle file?

```
test.maxHeapSize = ...
```

This way, we reduce the secret sauce we keep in CI configuration.